src/main/resources/db/migration/changes/UCSBDiningCommonsMenuItem.json

```diff
@@ -0,0 +1,59 @@
+{ "databaseChangeLog": [
+    {
+        "changeSet": {
+          "id": "UCSBDiningCommonsMenuItem-7",
+          "author": "RainaK",
+          "preConditions": [
+            {
+              "onFail": "MARK_RAN"
+            },
+            {
+              "not": [
+                {
+                  "tableExists": {
+                    "tableName": "UCSBDININGCOMMONSMENUITEMS"
+                  }
+                }
+              ]
+            }
+          ],
+          "changes": [
+            {
+              "createTable": {
+                "columns": [
+                  {
+                    "column": {
+                      "constraints": {
+                        "nullable": false
```

Comment:
Is there a primary key name?

Reply:
i accidentally deleted it before i made my pr. just fixed it!